provider.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "3.105.0"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {}
+}
```

Comment:
![image](https://github.com/user-attachments/assets/2aaf993b-a69f-43d8-9e51-8535b58b4971)
Add Eof(End-of-file), need to add blank line at the end